api/db-cdr/generate-cdr/make-bq-criteria-tables.sh

```diff
@@ -5830,13 +5829,12 @@ WHERE REGEXP_CONTAINS(name, r'[\"]')"
 
 
 ###############################################
-# FULL_TEXT and SYNONYMS
+# FULL_TEXT
 ###############################################
-echo "FULL_TEXT and SYNONYMS - adding data"
+echo "FULL_TEXT - adding data"
 bq --quiet --project_id=$BQ_PROJECT query --nouse_legacy_sql \
 "UPDATE \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\` x
 SET   x.full_text = y.full_text
-    , x.synonyms = y.full_text
```

Comment:
I was not able to test. Once its merged in i am planning to run the entire build for DummySr Project to confirm.

Reply:
I found generate-cb-criteria-tables command running it now on my local to test the changes